recipe/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "fitter" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fitter-{{ version }}.tar.gz
+  sha256: d9df8885f106d911c3a7ed97949b3cf3c29ec2464c529244f2434738dd6b0e50
+
+build:
+  entry_points:
+    - fitter = fitter.main:main
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - poetry-core
+    - pip
+  run:
+    - python
+    - click >=8.1.6,<9.0.0
+    - joblib >=1.3.1,<2.0.0
+    - matplotlib-base >=3.7.2,<4.0.0
```

Comment:
Why is`matplotlib-base` used instead of `matplotlib` ?

Reply:
@msentissi it's because `matplotlib-base` is a lighter package for us